package.json

```diff
@@ -7,13 +7,25 @@
     "build": "next build",
     "start": "next start"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "src/**/*.{ts,css,html}": [
```

Comment:
Could you rewrite this section to something like:

```diff
"lint-staged": {
-  "src/**/*.{ts,css,html}": [
+    "src/**/*.{tsx,ts}": [
     "next lint,
+    "prettier --write"
   ],
+  "**/*.{json,md,yml}": "prettier --write"
},
```

Everything else is :100: right now, thanks for recreating a fresh PR.

Explanation about the changes I requested:

1. We're using TypeScript which is why we need to lint `.tsx` & `.ts` files.
2. Prettier will standardise un-uniform coding practices at at "commit-time".

cc: @varshney-himanshu could you confirm the `prettier` commands we implemented here are correct?

Reply:
Sure will do !